src/GenericRestClient.ts

```diff
@@ -8,82 +8,87 @@
 
 import _ = require('lodash');
 import SyncTasks = require('synctasks');
-
-import { SimpleWebRequest, WebRequestOptions, WebResponse } from './SimpleWebRequest';
+import {
+    WebRequestOptions,
+    WebResponse,
+    Headers,
+    SimpleWebRequest,
+} from './SimpleWebRequest';
 
 export type HttpAction = 'POST'|'GET'|'PUT'|'DELETE'|'PATCH';
 
 export interface ApiCallOptions extends WebRequestOptions {
-    backendUrl?: string;
     excludeEndpointUrl?: boolean;
+    backendUrl?: string;
```

Comment:
Nothing special, all three props are optional, and Editor(_I'll turn off this option for the current project in order to avoid such situation_) suggests to move the longest property to first place in order to improve readability:

```typescript
export interface ApiCallOptions extends WebRequestOptions {
    excludeEndpointUrl?: boolean;
    backendUrl?: string;
    eTag?: string;
}
```